src/pages/Registration/Steps/Documentation/types.ts

```diff
@@ -1,3 +1,8 @@
 import { Documentation } from "pages/Registration/types";
 
-export type FormValues = Omit<Documentation, "level"> & { level: never };
+type Key = keyof Documentation;
+const _level: Key = "level";
+
+export type FormValues = Omit<Documentation, typeof _level> & {
```

Comment:
Field `level` redundant. 
```ts
export type FormValues = Omit<Documentation, typeof _level> & {
 [_level]: never;
};
```
can be:
```ts
export type FormValues = Omit<Documentation, typeof _level>;
```

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1789/commits/422182ed8381409fe2d2b8c41a7a1a5a9e44b216